.dockerignore

```diff
@@ -0,0 +1,3 @@
+.git
+.env
+.dockerignore
```

Comment:
Looked at the docs. Still don't know why the file `.dockerignore` includes the line `.dockerignore`.